fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -489,14 +492,27 @@ private void handleShowMigrations() throws AnalysisException {
     }
 
     // Show databases statement
-    private void handleShowDb() throws AnalysisException {
+    private void handleShowDb() throws AnalysisException, DdlException {
         ShowDbStmt showDbStmt = (ShowDbStmt) stmt;
         List<List<String>> rows = Lists.newArrayList();
-        List<String> dbNames = ctx.getGlobalStateMgr().getClusterDbNames(ctx.getClusterName());
+        List<String> dbNames = new ArrayList<>();
+
+        if (showDbStmt.getCatalogName() == null) {
+            dbNames = ctx.getGlobalStateMgr().getClusterDbNames(ctx.getClusterName());
```

Comment:
what's the different between getClusterDbNames and localMetaStore.listDbNames @stephen-shelby

Reply:
Both are currently the same. you could ref LocalMetastore.createDb() and unprotectCreateDb()

Reply:
There is now only one default cluster instance which named 'default_cluster'

Reply:
how about merge this 'if-else' block and just use connectorMetadata.get().listDbNames();?

Reply:
It is better to keep the previous usage, and then modify the corresponding call when the cluster is completely deleted.

Reply:
> It is better to keep the previous usage, and then modify the corresponding call when the cluster is completely deleted.

done.